Puck/events.md

```diff
@@ -0,0 +1,118 @@
+# Puck events
+Puck sends a multitude of events that describe the user journey.
+
+## Enforced events
+By default, the Puck Client sends events related to the users session and page views.
+
+**visit**: This event is fired once per session, and marks the start of a new session.
+
+**view**: This event is fired every time a user visits a page.
```

Comment:
maybe rephrase to "every time a user views a page" so less confusing with *visit* item above...